lib/src/util/hoc.dart

```diff
@@ -45,6 +45,6 @@
 ///   ),
 /// ])(_$BigBlock);
 /// ```
-R Function(A) composeHocs<R, A extends R>(Iterable<R Function(A)> functions) {
+R Function(R) composeHocs<R>(Iterable<R Function(R)> functions) {
```

Comment:
This change fixes a bad implicit cast in the implementation and shouldn't affect consumers.

Thankfully, there are no consumers, so we know it won't affect any 😄 : https://sourcegraph.wk-dev.wdesk.org/search?q=composeHocs&patternType=literal